src/components/node-list/node-list-row-list.js

```diff
@@ -4,6 +4,20 @@ import { connect } from 'react-redux';
 import NodeListRow, { nodeListRowHeight } from './node-list-row';
 import LazyList from '../lazy-list';
 
+// modify display of labels for modular pipelines to show nested relationship
+// note this label indentation could be subject to further design changes
+const getItemLabel = (item) => {
+  const layer = '・';
+  const whiteSpace = '&nbsp;&nbsp;&nbsp;&nbsp;';
+  if (item.type === 'modularPipeline') {
+    // parse depth of modular pipeline from namespace(i.e id)
+    const levels = item.id.match(/\./g) ? item.id.match(/\./g).length : 0;
+
+    return whiteSpace.repeat(levels) + layer + item.highlightedLabel;
+  }
+  return item.highlightedLabel;
+};
+
```

Comment:
The dots appear for all levels now, not just nested ones:
![image](https://user-images.githubusercontent.com/1155816/114855950-65bb3400-9dde-11eb-8f4d-275b0dcde322.png)

Suggested fix:
```suggestion
// Modify display of labels for modular pipelines to show nested relationship.
// Note: This label indentation could be subject to further design changes.
const getItemLabel = (item) => {
  if (item.type === 'modularPipeline') {
    // Parse depth of modular pipeline from namespace(i.e id)
    const levels = item.id.match(/\./g) ? item.id.match(/\./g).length : 0;
    const whiteSpace = '&nbsp;&nbsp;&nbsp;&nbsp;';
    const layer = levels ? '・' : '';

    return whiteSpace.repeat(levels) + layer + item.highlightedLabel;
  }
  return item.highlightedLabel;
};

```